### PR TITLE
Feature/admin home

### PR DESCRIPTION
**Pull request by jonahjtr**
## Description

created admin home on home immediately after logging in to okta, now we can see the button to go to admin home, and start creating components in admin role

Fixes # (issue)

## Loom Video
loom video link [**_**here**_**](https://www.loom.com/share/609cf4e917de47eba7b808769f4bb68d)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/adminHome/index.js [hunk 1]
```diff
@@ -6,13 +6,15 @@ import { Layout } from 'antd';
 import Banner from '../../common/Banner';
 
 const { Content } = Layout;
-const adminHome = () => {
+
+const AdminHome = () => {
+  console.log('im in admin');
```
Comment (Ryan-E-Mark) [inline src/components/pages/adminHome/index.js:11]: Don't forget to remove the console.log

**Review by Ryan-E-Mark — CHANGES_REQUESTED** [review]
Starting to come together, great work! This brings us closer to getting the functionality we need across the multiple user roles. There's a console.log left in the index.js file within adminHome that should be removed, everything else is awesome.

File: src/components/pages/adminHome/adminSidebar.js [hunk 2]
```diff
@@ -43,23 +42,23 @@ function AdminSidebar() {
...
         </Menu.Item>
 
         <Menu.Item key="5" icon={<DesktopOutlined />}>
-          <a href="#">Courses</a>
+          <a href="!#">Courses</a>
         </Menu.Item>
 
         <Menu.Item key="6" icon={<PlusOutlined />}>
-          <a href="#">Add Courses</a>
+          <a href="!#">Add Courses</a>
```
Comment (RoseGabriely) [inline src/components/pages/adminHome/adminSidebar.js:61]: What is the purpose of these !s?

File: src/components/pages/adminHome/index.js [hunk 3]
```diff
@@ -6,13 +6,15 @@ import { Layout } from 'antd';
...
   return (
     <div>
       <Layout>
-        <AdminSidebar />
         <Content>
           <Banner />
+          <AdminSidebar />
           <div className="calendar" data-testid="calendar">
             {/*Calendaer still set to Instructor Props*/}
```
Comment (RoseGabriely) [inline src/components/pages/adminHome/index.js:19]: Little typo in "calendar" here!

File: src/components/pages/adminHome/index.js [hunk 4]
```diff
@@ -6,13 +6,15 @@ import { Layout } from 'antd';
...
             <AdminCalender />
```
Comment (RoseGabriely) [inline src/components/pages/adminHome/index.js:20]: Another "calendar" typo here!

File: src/components/pages/InstructorHome/index.js [hunk 5]
```diff
@@ -10,8 +10,8 @@ const InstructorHome = () => {
   return (
     <div>
       <Layout>
-        <InstructorSidebar />
         <Content>
+          <InstructorSidebar />
           <Banner />
           <div className="calendar" data-testid="calendar">
             <InstructorCalender />
```
Comment (RoseGabriely) [inline src/components/pages/InstructorHome/index.js:17]: "calendar" typo here!

**Review by RoseGabriely — APPROVED** [review]
Thanks for these additions! Looks like whoever was working on this last made some typos with the word "calendar", feel free to fix them with this PR, or it could be a separate PR.

**Review by Ryan-E-Mark — APPROVED** [review]

**Review by ElchinNasirov — APPROVED** [review]

File: src/components/pages/adminHome/adminSidebar.js (hunk 2 above)
Comment (paulstgermain) [reply src/components/pages/adminHome/adminSidebar.js:61]: In short, these are preventing the browser from moving your view to the top of the page when you click these currently empty links. 🙂

**Comment by paulstgermain** [comment]
When running this branch locally, the calendar renders well underneath the cutoff for the sidebar in our `AdminDashboard` component. We'll want to fix this moving forward (great task for any of our UX folks!), but I'm merging this PR as it's a great platform upon which to make that fix!

**Pull request merged**